testfiles/1ZoneUncontrolled_Win_ASH55_Thermal_Comfort.idf

```diff
@@ -597,7 +597,7 @@
 
   Lights,
     SPACE1-1 Lights 1,       !- Name
-    ZONE ONE,                !- Zone or ZoneList Name
+    ZONE ONE,                !- Zone or ZoneList or Space or SpaceList Name
```

Comment:
This field name change needs to be described in Rules22-1-0-to-22-2-0.md, enough on this topic, you get the idea.